build/hooks/40-set_default_terraform_version.sh

```diff
@@ -9,7 +9,11 @@ source /usr/local/bin/common.sh
 current_version=$(cat /home/easy_infra/.tfenv/version)
 output_log="/var/log/$(basename "${BASH_SOURCE[0]%%.sh}.log")"
 
-if [[ "${TERRAFORM_VERSION}" == "${current_version}" ]]; then
+if [[ -z "${TERRAFORM_VERSION}" ]]; then
```

Comment:
This is a (very) minor bugfix.  Without it, if you set `TERRAFORM_VERSION` to an empty string, you will get a "Unable to install Terraform , continuing" message.  The reason why is because if you run `tfenv install` without an argument (i.e. `TERRAFORM_VERSION` is empty string), it exits 0.